### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by savikkirill3**

### CODE REVIEW

**Comment by eugenekhristo** [comment]
2018-11-22_08h59_49

File: package-lock.json [hunk 1]
```diff
@@ -25,50 +25,50 @@
...
-        "semver": "^5.5.0",
-        "source-map-loader": "^0.2.3",
-        "source-map-support": "^0.5.0",
-        "stats-webpack-plugin": "^0.6.2",
-        "style-loader": "^0.21.0",
-        "stylus": "^0.54.5",
-        "stylus-loader": "^3.0.2",
-        "tree-kill": "^1.2.0",
-        "uglifyjs-webpack-plugin": "^1.2.5",
-        "url-loader": "^1.0.1",
```
Comment (eugenekhristo) [inline package-lock.json]: What's happened in package-lock.json???

**Review by eugenekhristo — COMMENTED** [review]
Just a bit interested.

File: package-lock.json (hunk 1 above)
Comment (savikkirill3) [reply package-lock.json]: I do not know

**Pull request merged**